applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/app_ZRelease.conf

```diff
@@ -25,8 +25,6 @@ CONFIG_ML_APP_ML_STATE_CONTROL_BUTTON_ID=0x0000
 
 CONFIG_CAF=y
 
-CONFIG_CAF_INIT_LOG_SENSOR_EVENTS=n
```

Comment:
Btw. could you disable these init log Kconfig options also for remaining release configurations? (for consistency)